### ISSUE TITLE
Command metadata shouldn't be part of event "data" payload

### ISSUE THREAD

**Issue by graste** [honeybee/honeybee#59]
The command metadata is at the moment put into the data payload of the events. It might be better to have `data` and `metadata` on the event and then let the projection update take care of maybe merging the `metadata` field info on the projection to have metadata available in the projections. The metadata key in the projection also might conflict with an actual schema property `metadata` if I see this correctly.

**Comment by MrHash** [comment]
It appears to me to be separate: https://github.com/honeybee/honeybee/blob/master/src/Infrastructure/Event/Event.php#L50

**Comment by graste** [comment]
Yeah I was thinking of this: https://github.com/honeybee/honeybee/blob/master/src/Infrastructure/DataAccess/Storage/CouchDb/EventStream/EventStreamAppender.php which might be different dependent on storage technology used.

**Comment by graste** [comment]
Still the question of the metadata in the projection as a "normal" properties remains.

**Comment by MrHash** [comment]
Well in the event store the metadata is a separate field to the data, but in the projection the metadata is stored inside the document, but then so are all the default attributes. I guess it depends on the technology how it is stored? otherwise the non-default attributes should be stored inside a `data` field but that might be confusing when querying etc.
